### ISSUE TITLE
Gnome-Builder crashes while trying to run project

### ISSUE THREAD

**Issue by onny** [NixOS/nixpkgs#219669]
### Describe the bug

Gnome-Crashes when trying to run a Python GTK4 project

### Steps To Reproduce
1. Create an example project with Python and GTK4
2. SDKs are already installed
3. Build went fine
4. Try to run project, Gnome-Builder crashes

### Expected behavior
Project should build and run

### Additional context
Error in terminal log
```
GLib-GIO[ 356849]:    ERROR: Settings schema 'org.gtk.Settings.Debug' is not installed
fish: Job 1, 'gnome-builder' terminated by signal SIGTRAP (Trace or breakpoint trap)
```

### Notify maintainers
@jtojnar @bobby285271

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.2.0, NixOS, 23.05 (Stoat), 23.05.20230301.a08d697`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.13.3`
 - channels(root): `"nixos-22.11"`
 - channels(onny): `"nixos-22.11"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`
```

**Comment by jtojnar** [comment]
Thanks for reporting, opened https://gitlab.gnome.org/GNOME/gnome-builder/-/merge_requests/712

**Comment by jtojnar** [comment]
This should be fixed by gnome-builder 44.0
